app/main.go

```diff
@@ -34,6 +35,9 @@ import (
 	"google.golang.org/appengine/log"
 )
 
+// Global Storage Manager object to be used while running the app
+var StorageManager storage.Manager
```

Comment:
This should be `storageManager` as it isn't (and can't) be exported from main.

Reply:
Acknowledged. Fixed.